From 015a94e0b7c8618bf3c4c80d4464ad27427c9420 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Tue, 7 Jul 2026 22:56:50 -0400
Subject: [PATCH] version: bump EXTENDNO to Reaper_v1.1-beta2

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 44e62e2728..b0385557f2 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.1-beta1
+EXTENDNO=Reaper_v1.1-beta2
 
 RCNO=0
-- 
2.25.1

