From b3238e47b5c2e0acb7fd3651648006f5ddf20a51 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Thu, 9 Jul 2026 08:56:11 -0400
Subject: [PATCH] version: Reaper_v1.2.1 (post-login dashboard landing fix)

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index a4cffd241b..e8b34ccc8f 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.2
+EXTENDNO=Reaper_v1.2.1
 
 RCNO=0
-- 
2.25.1

