From d4439cdc0e74cf70be88b94c579670b38daed8f9 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 25 Jul 2026 12:18:32 -0400
Subject: [PATCH 226/226] version bump EXTENDNO to Reaper_v1.7.8

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 6741510284..a1587d13ce 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.7.7
+EXTENDNO=Reaper_v1.7.8
 
 RCNO=0
-- 
2.25.1

