From 5409676d85c6f2c968034992fd013282b0168642 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Tue, 28 Jul 2026 09:43:59 -0400
Subject: [PATCH 247/248] be96u-only: v1.8.6b (i18n quote-safety fix)

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 5e334ed9ad..e1ba022285 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.8.6a
+EXTENDNO=Reaper_v1.8.6b
 
 RCNO=0
-- 
2.25.1

