From 8be215052030ad8c2504735d54f2490b40854af2 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Wed, 19 Aug 2026 11:41:10 -0400
Subject: [PATCH] version: bump EXTENDNO to Reaper_v2.5.0

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index c4428b4475..b1eed4d8fb 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.4.9
+EXTENDNO=Reaper_v2.5.0
 
 RCNO=0
-- 
2.25.1

