From cda82d28d90c6f18bc535d52f4c9da677ae211cf Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Fri, 21 Aug 2026 07:22:11 -0400
Subject: [PATCH] version: bump EXTENDNO to Reaper_v2.5.9

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 8a6acf745b..5ea78c9257 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.5.8
+EXTENDNO=Reaper_v2.5.9
 
 RCNO=0
-- 
2.25.1

