From b5224ceb473ea57afe63ccc51b789abb04944768 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Fri, 21 Aug 2026 13:11:13 -0400
Subject: [PATCH] version: bump EXTENDNO to Reaper_v2.6.0

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 5ea78c9257..fcf7bd749f 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.5.9
+EXTENDNO=Reaper_v2.6.0
 
 RCNO=0
-- 
2.25.1

