From 97228322f549a6fcaf98621319edcfcca006bf28 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Fri, 21 Aug 2026 21:47:39 -0400
Subject: [PATCH] version: Reaper_v2.6.1

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index fcf7bd749f..3118c19e33 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.6.0
+EXTENDNO=Reaper_v2.6.1
 
 RCNO=0
-- 
2.25.1

