From 3a8b5e5d1bcc38e74a48168a6342d5a99d930180 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 22 Aug 2026 09:53:33 -0400
Subject: [PATCH] version: Reaper_v2.6.4

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 54a9f65121..e33e1d3221 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.6.3
+EXTENDNO=Reaper_v2.6.4
 
 RCNO=0
-- 
2.25.1

