From 533f882448a0f01475f2c145860c1e57f9e22237 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 22 Aug 2026 11:43:19 -0400
Subject: [PATCH] version: Reaper_v2.6.6

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 3278b4383e..19c6fdd634 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.6.5
+EXTENDNO=Reaper_v2.6.6
 
 RCNO=0
-- 
2.25.1

