From e28eea2b87b74b38f586f76468b53febf0bd2f06 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 22 Aug 2026 12:41:26 -0400
Subject: [PATCH] version: Reaper_v2.6.7

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 19c6fdd634..5f23011f7c 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.6.6
+EXTENDNO=Reaper_v2.6.7
 
 RCNO=0
-- 
2.25.1

