From f446a7d481fef51237f74eaed351196e74babec3 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 22 Aug 2026 13:09:49 -0400
Subject: [PATCH] version: Reaper_v2.6.8

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 5f23011f7c..731751c2f0 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.6.7
+EXTENDNO=Reaper_v2.6.8
 
 RCNO=0
-- 
2.25.1

