From 022c7e7145f608247c77a4dc2fc1329458948221 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 22 Aug 2026 16:09:28 -0400
Subject: [PATCH] version: Reaper_v2.7.1

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 07cb563e1c..9055a068af 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.7.0
+EXTENDNO=Reaper_v2.7.1
 
 RCNO=0
-- 
2.25.1

