From d512414d725017be0e3bd611aba55803c24373ec Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sun, 23 Aug 2026 06:33:54 -0400
Subject: [PATCH] version: Reaper_v2.7.3

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 2a1e551d5d..222941c727 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.7.2
+EXTENDNO=Reaper_v2.7.3
 
 RCNO=0
-- 
2.25.1

