From c7635b7f73c4c8de12e499465ff35e422389f5aa Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sun, 23 Aug 2026 18:04:08 -0400
Subject: [PATCH] version: Reaper_v2.7.4

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 222941c727..b51943c4d2 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.7.3
+EXTENDNO=Reaper_v2.7.4
 
 RCNO=0
-- 
2.25.1

