From 8080ad111cd40f2002df9880a8826f88b3bca8a2 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sun, 23 Aug 2026 20:02:02 -0400
Subject: [PATCH] version: Reaper_v2.7.5

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index b51943c4d2..f2eae04aec 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.7.4
+EXTENDNO=Reaper_v2.7.5
 
 RCNO=0
-- 
2.25.1

