From 71a4fd46ca3ab133adbdc5f422ea4f32833e4394 Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sun, 23 Aug 2026 22:11:14 -0400
Subject: [PATCH] version: Reaper_v2.7.6

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index f2eae04aec..6d279f267e 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.7.5
+EXTENDNO=Reaper_v2.7.6
 
 RCNO=0
-- 
2.25.1

